mule-migration-tool-library/src/main/java/com/mulesoft/tools/migration/library/mule/steps/core/MigrateApiConfigFlows.java

```diff
@@ -0,0 +1,210 @@
+/*
+ * Copyright (c) 2020, Mulesoft, LLC. All rights reserved.
+ * Use of this source code is governed by a BSD 3-Clause License
+ * license that can be found in the LICENSE.txt file.
+ */
+package com.mulesoft.tools.migration.library.mule.steps.core;
```

Comment:
it must be a post migration step because during the API Kit migration the ApplicationModel is not yet saved. Also the UriParam transformation must be the first MuleEvent in a flow for the metadata to work.